src/integrationTest/java/uk/gov/hmcts/darts/event/controller/DarNotifyControllerTest.java

```diff
@@ -56,14 +64,50 @@ void setup() {
 
     @Test
     void shouldSendDarNotifyEventSoapAction() throws Exception {
-        darPcStub.respondWithSuccessResponse();
+        darPcStub.respondWithSuccessResponse(OffsetDateTime.now());
```

Comment:
If we can have an injectable component to set the "current time" in `DarPcTimeLogInterceptor`, then we could explicitly set the current time to a fixed value, and create a response with a fixed Date header value. I think that approach would be more explicit and potentially less flaky than relying on the static `OffsetDateTime.now()` calls (and also more in line with how we test these things in darts-api).

Reply:
I have updated the code to use clocks to allow us to set fixed times